src/open_inwoner/accounts/signals.py

```diff
@@ -47,14 +47,14 @@ def update_user_on_login(sender, user, request, *args, **kwargs):
         try:
             service = OpenKlant2Service()
         except Exception:
-            logger.error("OpenKlant2 service failed to build")
+            logger.warning("OpenKlant2 service failed to build")
```

Comment:
Let's also log the exception info, so if we are reading the logs we can see what the error in lieu of finer-grained exceptions):

```suggestion
            logger.warning("OpenKlant2 service failed to build", exc_info=True)
```